GiTinder/Services/UserServices.cs

```diff
@@ -12,6 +27,46 @@ public UserServices(GiTinderContext context)
         {
             _context = context;
         }
+<<<<<<< HEAD
+        //async method is used everytime something should take long time like for requesting for data await allows certain line to run independently.
+        //if user doesn't exist in github then it shouldnt respond
+        public async Task<User> GetUserAsync(string username)
```

Comment:
@Riceqrisp name this method like GetGithubProfileAsync

Reply:
Re-named methods